lib/util.js

```diff
@@ -38,7 +38,9 @@ const loggers = {
   network: global.COM_RS_COMMON_LOGGER.makeComponentLogger("_zsf.network"),
   langManager: global.COM_RS_COMMON_LOGGER.makeComponentLogger("_zsf.lang"),
   clusterLogger: global.COM_RS_COMMON_LOGGER.makeComponentLogger("_zsf.cluster"),
-  storeLogger: global.COM_RS_COMMON_LOGGER.makeComponentLogger("_zsf.store")
+  storeLogger: global.COM_RS_COMMON_LOGGER.makeComponentLogger("_zsf.store"),
+  syncLogger: global.COM_RS_COMMON_LOGGER.makeComponentLogger("_zsf.sync"),
+  raftLogger: global.COM_RS_COMMON_LOGGER.makeComponentLogger("_zsf.raft"),
```

Comment:
remember to document in the github.com/zowe/zlux/wiki page.

Reply:
Sure.